src/ConnectedMode/Migration/ConnectedModeMigration.cs

```diff
@@ -141,10 +145,22 @@ private async Task MigrateImplAsync(BoundSonarQubeProject oldBinding, IProgress<
             // Trigger a re-fetch of suppressions so the Roslyn settings are updated.
             await suppressionIssueStoreUpdater.UpdateAllServerSuppressionsAsync();
 
+            if (shareBinding)
+            {
+                progress?.Report(new MigrationProgress(0, 1, "Saving Shared Binding Config...", false));
+                SaveSharedBinding(oldBinding);
+            }
+
             progress?.Report(new MigrationProgress(0, 1, "Migration finished successfully!", false));
             logger.WriteLine(MigrationStrings.Process_Finished);
         }
 
+        private void SaveSharedBinding(BoundSonarQubeProject oldBinding)
+        {
+            var sharedBindingConfigModel = new SharedBindingConfigModel { Organization = oldBinding.Organization?.Key, ProjectKey = oldBinding.ProjectKey, Uri = oldBinding.ServerUri.ToString() };
```

Comment:
NOTE: I don't like how we designed this SharedBindingModel.Uri thing, but it's too late now